source/languages/en/riak/dev/advanced/replication-properties.md

```diff
@@ -524,8 +524,8 @@ vnodes responsible for an object.
 Scenario | What happens in Riak
 :--------|:--------------------
 All 3 vnodes agree on the value | Once the first 2 vnodes return the value, that value is returned to the client
-2 of 3 vnodes agree on the value, and those 2 are the first to reach the coordinating node | The value is returned returned to the client. Read repair will deal with the conflict per the later scenarios, which means that a future read may return a different value or <a href="/latest/theory/concepts/vector-clocks#siblings">siblings</a>
-2 conflicting values reach the coordinating node and <a href="/latest/theory/concepts/vector-clocks">vector clocks</a> allow for resolution | The vector clocks are used to resolve the conflict and return a single value, which is propagated via read repair to the relevant vnodes
+2 of 3 vnodes agree on the value, and those 2 are the first to reach the coordinating node | The value is returned returned to the client. Read repair will deal with the conflict per the later scenarios, which means that a future read may return a different value or <a href="/theory/concepts/vector-clocks#siblings">siblings</a>
```

Comment:
Actually, these links should be both be changed to `/theory/concepts/context#Siblings`